topi/include/topi/transform.h

```diff
@@ -107,6 +107,36 @@ inline Tensor transpose(const Tensor& x,
     }, name, tag);
 }
 
+/*!
+* \brief reverse elements of an array in a particular axis
+*
+* \param x The input tensor
+* \param axis The axis along which the tensors will be reveresed
+* \param name The name of the operation
+* \param tag The tag to mark the operation
+*
+* \return A Tensor whose op member is the reverse operation
+*/
+inline Tensor reverse(const Tensor& x,
```

Comment:
If the name of function is flip in numpy, we should always be consistent with numpy instead of tensorflow